fern/pages/docs/introduction/project-structure.mdx

```diff
@@ -0,0 +1,129 @@
+---
+title: Project Structure
+excerpt: An overview of the file and folder structure of a Fern Docs project
+---
+
+This page provides an overview of the file and folder structure of a Fern Docs project.
+
+## Top-level Folders 
```

Comment:
I think we should explain that this is a format that we encourage but not required? Fileforge is an example where this format doesn't make sense